addons/account_peppol/models/account_move.py

```diff
@@ -13,6 +13,7 @@ class AccountMove(models.Model):
         selection=[
             ('ready', 'Ready to send'),
             ('to_send', 'Queued'),
+            ('skipped', 'Skipped'),
```

Comment:
You need a magic trick to do that in stable version like here:
https://github.com/odoo/enterprise/commit/10ec1990ba0c8621bf2804753aab1915479b935f#diff-8afc82e663ecf1ebc79d464ba7d9e96a7fda9fe2b431abecbd4f7216114de2dfR594